src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1,4 +1,14 @@
-// Add the required types and props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
+import React from 'react';
+import { Todo } from '../../types/Todo';
+import { UserInfo } from '../UserInfo';
+
+interface Props {
+  todo: Todo;
+}
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => (
+  <article className={`TodoInfo ${todo.completed ? 'TodoInfo--completed' : 'false'}`}>
+    <h2 className="TodoInfo__title">{todo.title}</h2>
+    {todo.user && <UserInfo user={todo.user} />}
+  </article>
 );
```

Comment:
Let's use destructuring assignment here
With this condition it still return `false` as classname. You don't need to add class false on negative comparison

![image](https://github.com/mate-academy/react_static-list-of-todos/assets/84296041/96077ec5-64a4-4c1d-a07a-d99e108a0aad)

```suggestion
export const TodoInfo: React.FC<Props> = ({ todo }) => {
  const {
    comleted,
    title,
    user
  } = todo;
 
  return (
    <article className={`TodoInfo ${todo.completed ? 'TodoInfo--completed' : ''}`}>
      <h2 className="TodoInfo__title">{todo.title}</h2>
      {todo.user && <UserInfo user={todo.user} />}
    </article>
  )
}
```